chia/rpc/data_layer_rpc_api.py

```diff
@@ -157,5 +157,9 @@ async def get_roots(self, request: Dict[str, Any]) -> Dict[str, Any]:
         # todo input checks
         if self.service is None:
             raise Exception("Data layer not created")
-        res = await self.service.get_roots(store_ids)
-        return {"hashes": res}
+        roots = []
+        for id in store_ids:
+            id_bytes = bytes32.from_hexstr(id)
+            res = await self.service.get_root(store_id=id_bytes)
+            roots.append({"id": id_bytes, "hash": res})
```

Comment:
Not a requested change, just for consideration.  How do we decide between `List[TypedDict]` vs just `Dict[str, str]`?  Like this could just be a mapping from the id to the hash.